src/testing/fixtures/algorand-fixture.ts

```diff
@@ -86,34 +72,36 @@ export function algorandFixture(fixtureConfig: AlgorandFixtureConfig | undefined
  * @param config The algo configuration
  * @returns The fixture
  */
+export function algorandFixture(fixtureConfig: AlgorandFixtureConfig | undefined, config: AlgoConfig): AlgorandFixture
+
 export function algorandFixture(fixtureConfig?: AlgorandFixtureConfig, config?: AlgoConfig): AlgorandFixture {
-  config = config || getConfigFromEnvOrDefaults()
+  fixtureConfig = { ...fixtureConfig, ...config }
+  if (!fixtureConfig.algod || !fixtureConfig.indexer || !fixtureConfig.kmd) {
+    fixtureConfig = { ...ClientManager.getConfigFromEnvironmentOrLocalNet(), ...fixtureConfig }
+  }
 
-  const algod = fixtureConfig?.algod ?? getAlgoClient(config.algodConfig)
-  const indexer = fixtureConfig?.indexer ?? getAlgoIndexerClient(config.indexerConfig)
-  const kmd = fixtureConfig?.kmd ?? getAlgoKmdClient(config.kmdConfig)
+  const algod = fixtureConfig.algod ?? ClientManager.getAlgodClient(fixtureConfig.algodConfig!)
+  const indexer = fixtureConfig.indexer ?? ClientManager.getIndexerClient(fixtureConfig.indexerConfig!)
+  const kmd = fixtureConfig.kmd ?? ClientManager.getKmdClient(fixtureConfig.kmdConfig!)
   let context: AlgorandTestAutomationContext
   let algorandClient: AlgorandClient
 
   const beforeEach = async () => {
     Config.configure({ debug: true })
     const transactionLogger = new TransactionLogger()
     const transactionLoggerAlgod = transactionLogger.capture(algod)
-    const acc = await getTestAccount(
-      { initialFunds: fixtureConfig?.testAccountFunding ?? algos(10), suppressLog: true },
-      transactionLoggerAlgod,
-      kmd,
-    )
     algorandClient = algorandClient ?? AlgorandClient.fromClients({ algod: transactionLoggerAlgod, indexer, kmd })
+    const acc = await getTestAccount({ initialFunds: fixtureConfig?.testAccountFunding ?? algos(10), suppressLog: true }, algorandClient)
     algorandClient.setSignerFromAccount(acc).setDefaultValidityWindow(1000).setSuggestedParamsTimeout(0)
     const testAccount = { ...acc, signer: algorandClient.account.getSigner(acc.addr) }
     context = {
+      algorand: algorandClient,
```

Comment:
Added this easier for better ergonomics if grabbing other stuff from the context.